### PR TITLE
Allow to download files linked to a bank operation

### PR DESCRIPTION
**Pull request by frankrousseau**
* When a file is linked to the bank operation, the amount turns into a link that allows to download that file.
* I removed the old icon to change it by the "Mes comptes" one. Because: 
     * We will soon remove the "Mes comptes application" because we want to remove Weboob from the Data Indexer.
     * The Mes comptes SVG  icon is less buggy than the Kresus SVG icon.

### CODE REVIEW

File: server/controllers/operations.coffee [hunk 1]
```diff
@@ -42,50 +42,73 @@ module.exports.delete = (req, res) ->
 
 module.exports.query = (req, res) ->
 
-    paramAccounts = req.body.accounts or [-1]
-
-    BankOperation.allFromBankAccounts paramAccounts, (err, operations) ->
-        if err?
-            res.send 500, error: 'Server error occurred while retrieving data'
+        paramAccounts = req.body.accounts or [-1]
```
Comment (bnjbvr) [inline server/controllers/operations.coffee]: I suspect you haven't changed anything to .query but the indentation, can you put it back to what it was? (and squash your commits, please)

File: server/controllers/operations.coffee [hunk 2]
```diff
@@ -42,50 +42,73 @@ module.exports.delete = (req, res) ->
...
-                    callback null, operation
-
-            # check all bank operations
-            async.concat operations, treatment, (err, results) ->
-                if err?
-                    errorMsg = 'Server error occurred while retrieving data'
-                    res.send 500, error: errorMsg
-                else
-                    res.send 200, results
+            res.status(404).send 'File not found'
```
Comment (bnjbvr) [inline server/controllers/operations.coffee]: Is the only possible error that the file isn't found? Could we just `res.status(stream.statusCode);` instead?

File: server/models/bankoperation.coffee [hunk 3]
```diff
@@ -8,6 +8,7 @@ module.exports = BankOperation = americano.getModel 'bankoperation',
     raw: String
     dateImport: Date
     categoryId: String
+    binary: (x) -> x
```
Comment (bnjbvr) [inline server/models/bankoperation.coffee]: IIUC cozy-data-system's code, binary is an object but it doesn't contain the file itself. Is that correct? If so, a comment would be appreciated here (for instance, an example of what a binary object could be).

**Comment by bnjbvr** [comment]
Super nice, thanks for doing this! Any chance you could address my comments, and propose to merge your PR into `incoming`, rather than `master`, please?

File: server/models/bankoperation.coffee (hunk 3 above)
Comment (frankrousseau) [inline server/models/bankoperation.coffee]: I don't know what IIUC means.

I will add a comment.

File: server/controllers/operations.coffee (hunk 2 above)
Comment (frankrousseau) [inline server/controllers/operations.coffee]: Ok. I will see how to figure that out.

File: server/controllers/operations.coffee (hunk 1 above)
Comment (frankrousseau) [inline server/controllers/operations.coffee]: I don't know what squashing is. I will try to do it but not sure about what you want.

File: server/models/bankoperation.coffee (hunk 3 above)
Comment (bnjbvr) [inline server/models/bankoperation.coffee]: IIUC == If I Understand Correctly. Thanks for adding the comment!

File: server/controllers/operations.coffee (hunk 1 above)
Comment (bnjbvr) [inline server/controllers/operations.coffee]: Squashing is the process of merging two commits into one, by modifying history (`git rebase -i`). It's not a big deal if you don't do it, no worries :)

**Pull request closed without merging**